client/src/containers/Home/Home.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import Carousel from "../../components/UI/Carousel/Carousel";
+import { connect } from "react-redux";
+import axios from "axios";
+import Category from "../../components/UI/Category/Category";
+
+function Home(props) {
+  const [banners, setBanners] = React.useState([]);
+  const [categories, setCategories] = React.useState([]);
+
+  React.useEffect(() => {
+    axios
+      .get("http://localhost:5001/banners")
+      .then((res) => setBanners(res.data))
+      .catch((err) => console.log(err));
+    axios
+      .get("http://localhost:5001/categories")
+      .then((res) => setCategories(res.data))
```

Comment:
Follow the same approach as mentioned above. Also dont directly hardcode url string. Use variables.

Reply:
Done